example/main.tf

```diff
@@ -62,7 +62,7 @@ variable "container_memory" {
 }
 
 variable "container_memoryreservation" {
-  default = "64"
+  default = "500"
```

Comment:
The Default container is an emty apache. It can have low ram. And this task can be replaced later. Or what this change for something else?